lang/sbcl/Portfile

```diff
@@ -39,7 +39,9 @@ patchfiles      0001-fix-building-when-root-directory-contain-non-ASCII-c.patch
 # https://github.com/sbcl/sbcl/pull/62
 patchfiles-append \
                 0003-runtime.c-fix-time-header-for-LISP_FEATURE_AVOID_CLO.patch
-
+# https://github.com/sbcl/sbcl/pull/64
+patchfiles-append \
+                0004-0004-fix-pthread-macOS.patch
```

Comment:
Typo, should be `0004-fix-pthread-macOS.patch`.

Reply:
Thank you! Will fix now.